app/views/internal/tools/index.html.erb

```diff
@@ -1,12 +1,12 @@
 <h2>General Purpose URL Purge</h2>
 <p>For a page that should 404. This will purge the page via Fastly.</p>
-<p>Use the full path (https://dev.to/xyz) or the relative path: (/xyz)</p>
+<p>Use the full path (<%= app_url('xyz') %>) or the relative path: (/xyz)</p>
 <div class="row m-3">
   <%= form_tag bust_cache_internal_tools_path do %>
     <div class="form-row">
       <div class="form-group">
         <%= label_tag("Link:") %>
-        <%= text_field_tag :dead_link, nil, placeholder: "/devteam/page" %>
+        <%= text_field_tag :dead_link, nil, placeholder: "/xyz" %>
         <%= submit_tag("Clear Cache") %>
```

Comment:
changed this from `/devteam/page` as there was no way to neat way to do this, and xyz seemes inline with the description